### ISSUE TITLE
NSC is recognised on Apple II/II+ when slot-3 is empty

### ISSUE THREAD

**Issue by tomcw** [AppleWin/AppleWin#827]
The NSC is a 28-pin chip, but on the Apple II/II+ the 6 ROM chips are all 24-pin (eg. 6x 2K = 12K ROM). See Sather's UTAII, Ch.6 (page 6-1)... so this means that NSC can only be supported via a peripheral card with a 28-pin/4K ROM (but does such a card exist? since $C800 space is only 2K).

Repro:
- Select Model: Apple II or II+
- Config->Ethernet settings... and set Ethernet=Disabled
- in VS, set a breakpoint after a successful call to `IsPotentialNoSlotClockAccess()`
- drop to the Monitor, then read C300
- the BP will trigger

So the fix is to disallow NSC for II/II+ models.

NB. the BP triggers because in `IsPotentialNoSlotClockAccess()` this is true:
```
	((SW_INTCXROM || !SW_SLOTC3ROM) && (AddrHi == 0xC3))
```
But these soft-switches don't exist on the II/II+.

**Comment by tomcw** [comment]
John Brooks pointed out that:
>NSC is available for 24-pin or 28-pin. See: https://manilagear.com/product/no-slot-clock/
>The Dallas 1216 clock only uses 2 pins, A0 and A2, so doesn't really care about the size of the ROM (or dram for DClock) that it is under.

From the Manila Gear manual, the NSC (24-pin config) does work on the II/II+, eg. when piggybacking the SSC's 24-pin ROM.

Anyway, the bug is still valid. To support NSC for the II/II+, we'd need:
- to support the NSC via the external SSC ROM (this would also work with the //e)
- or a ProDOS driver that eg. supported internal F8 ROM (for II/II+ only)

**Comment by tomcw** [comment]
Info:
- No-Slot Clock Utilities v1.2 (C)1990 initially writes to $C300 just before reading the NSC's time (or setting the time). See $2A23 and $2AA2 respectively, eg:
```
2A23: STA $C300
      LDA $C304
```

- ProDOS8 2.5.0a8 doesn't write to $C300 (for NoSlotClk.IIe). Just before reading the NSC's time it only reads $C304 (I've no idea about setting the time), eg:
```
D74B: BIT $C304
```

AIUI, these initial accesses to $C300/C304 are to reset the NSC's comparitor. Presumably the P8 2.5.0a8 team think there's no need for the initial write to $C300.

Same for NoSlotClk.II (there's no write to $F8xx):
```
0213: BIT $F804
```

So in terms of supporting NSC for the Apple II/II+ (at $F800), then perhaps there no need to support **writes** to F800 I/O (ie. a simplification to the AppleWin implementation).

**Comment by tomcw** [comment]
Released [AppleWin 1.30.0.0](https://github.com/AppleWin/AppleWin/releases/tag/v1.30.0.0(try2)), so closing this.
